src/xopen/__init__.py

```diff
@@ -383,9 +384,9 @@ class PipedGzipReader(PipedCompressionReader):
     """
     def __init__(self, path, mode: str = "r", threads: Optional[int] = None):
         try:
-            super().__init__(path, "pigz", mode, "-p", threads)
+            super().__init__(path, ["pigz"], mode, "-p", threads)
```

Comment:
We should probably (in a different PR) split this up into a `PipedGzipReader` and a `PipedPigzReader` (same for the `...Writer`) because that would be more consistent with the other classes: They run the tool or fail, and any fallback to a different tool is handled outside the class.